### PR TITLE
[EMB-433] Fix: registries navbar bugs

### PR DESCRIPTION
**Pull request by fabmiz**
## Purpose

Add. dev. fixes to adress issues and bugs reported in QA testing

## Summary of Changes
On mobile:

- Use services full name `OSF Registries` instead of `Registries`
- Show all links in a dropdown
- Show `donate` and `help` links on mobile

On tablet:
- Firefox: fix issue with help link overlapping with search input.

## Side Effects

N/A

## Feature Flags

`ember_registries_detail_page`

## QA Notes

Please see if all issues reported in QA testing are addressed.

## Ticket

[EMB-433](https://openscience.atlassian.net/browse/EMB-433)

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

**Comment by aaxelb** [comment]
Just one thing: Could you style "logout" the same as the other dropdown options?

**Comment by fabmiz** [comment]
@aaxelb fixed.

File: lib/registries/addon/components/registries-navbar/template.hbs [hunk 1]
```diff
@@ -101,43 +102,53 @@
...
-            {{else}}
-
-                <BsDropdown
-                    data-analytics-scope='User dropdown'
-                    local-class='AuthDropdown'
-                    as |dd|
+            <BsDropdown
+                data-analytics-scope='User dropdown'
+                local-class='AuthDropdown'
+                as |dd|
```
Comment (herman-rogers) [inline lib/registries/addon/components/registries-navbar/template.hbs]: Please use a more descriptive name than "dd", it's hard to tell what context this is apart of.

File: lib/registries/addon/components/registries-navbar/template.hbs [hunk 2]
```diff
@@ -101,43 +102,53 @@
...
+                            {{nav.icon (if dd.isOpen 'times' 'bars')}}
+                        </OsfButton>
+                    {{/if}}
+                </dd.toggle>
+
+                <dd.menu
+                    data-test-auth-dropdown
+                    class='dropdown-menu-right'
+                    local-class='AuthDropdownMenu DropdownMenu'
+                    as |ddm|
```
Comment (herman-rogers) [inline lib/registries/addon/components/registries-navbar/template.hbs]: same thing here - maybe just "menu"?

**Pull request merged**